resources/js/components/Committee.vue

```diff
@@ -0,0 +1,65 @@
+<script>
+import CommitteeInfo from './CommitteeInfo';
+import CommitteeMemberList from './CommitteeMemberList';
+import CommitteeSessions from './CommitteeSessions';
+export default {
+    name: 'Committee',
+    props: {
+        info: {
+            type: String,
+            default: () => 'info'
+        },
+        members: {
+            type: Array,
+            default: () => []
+        },
+        meetings: {
+            type: Array,
+            default: () => []
+        }
+    },
+    components: {
+        CommitteeInfo,
+        CommitteeMemberList,
+        CommitteeSessions
+    },
+    data() {
+        return {
+            activeTab: 'mitglieder'
+        };
+    },
+};
+</script>
+
+<template>
+    <div class="">
+        <nav class="ris-committee-navigation ris-without-padding-mob">
+            <ul class="ris-ul ris-committee-navigation__list">
+                <li class="ris-committee-navigation__item">
+                    <button
+                        @click="activeTab = 'aufgaben'"
+                        :class="['ris-committee-navigation__button ris-committee-navigation__button--aufgaben', activeTab === 'aufgaben' ? 'active' : '' ]">
```

Comment:
i also recommend to move static classes out of `:class`, so just put `ris-committee-navigation__button ris-committee-navigation__button--aufgaben` in `class`